module/x/auction/keeper/keeper.go

```diff
@@ -26,15 +26,23 @@ type Keeper struct {
 }
 
 // GetParams get params
-// GetParams returns the parameters from the store
-func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
-	k.paramSpace.GetParamSet(ctx, &params)
-	return
+func (k Keeper) GetParams(ctx sdk.Context) (types.Params, error) {
```

Comment:
should revert this. Gravity bridge still use sdk 45.16 so the params should still store on the paramspace